lib/QubitFlatfileImport.class.php

```diff
@@ -1206,7 +1206,7 @@ protected function rowProcessingBeforeSave($row)
         // otherwise, if column is data and a handler for it is set, use it
         call_user_func_array(
           $self->handlers[$columnName],
-          array($self, $value)
+          array(&$self, $value)
```

Comment:
I don't understand why in first place this function prefers the object to be passed by reference if we already get a handle by default. I think I would try instead to update the func declaration to `function($self, $index, $columnName, $value)` and leave the array declaration as `array($self, $value)`. That would make the code much easier to understand IMO. More here: http://php.net/manual/en/language.oop5.references.php.